PoE-Overlay-Community/PoE-Overlay-Community-Fork
# Updated the poe assets

## Description

This branch only contains updated PoE assets created using the PoE Asset Updater at commit https://github.com/WhiteFang5/PoE-Asset-Updater/commit/622bb2c1e422b19cccbac08da54dea255ced0362

This PR fixes https://github.com/PoE-Overlay-Community/PoE-Overlay-Community-Fork/issues/3 (being harvest league ready)

This PR fixes https://github.com/PoE-Overlay-Community/PoE-Overlay-Community-Fork/issues/66 (missing annointments)

This PR fixes https://github.com/PoE-Overlay-Community/PoE-Overlay-Community-Fork/issues/80 (Incorrect/unkwnon stat_######)

This PR blocks https://github.com/PoE-Overlay-Community/PoE-Overlay-Community-Fork/pull/70 (metamorph sample evaluation)

This PR blocks https://github.com/PoE-Overlay-Community/PoE-Overlay-Community-Fork/pull/71 (Vials, Delirium Orbs and Scarabs price overview)

## How to help test this branch
- Run it in dev mode, report any issues related to the assets
- Hop into Any League and go price check a bunch of items, report any issues regarding specific stats/mods not working or being incorrect.

## How Has This Been Tested?

- [ ] ran `npm run ng:lint`
- [ ] ran `npm run format`
- [ ] ran `npm run ng:test`
